scripts/generate-catalog.py

```
@@ -3,22 +3,34 @@
+import re
+from datetime import timedelta, datetime
 
 from jinja2 import Environment
 import git
 from jinja2 import Environment, FileSystemLoader, select_autoescape
 import yaml
 
-from common import store_data, check_repo_exists, call_rate_limit_aware, g, previous_repos, previous_skips, blacklist, snakefmt_version, offset
+from common import (
+    store_data,
+    check_repo_exists,
```

Ruff (0.8.2)

`common.check_repo_exists` imported but unused

Remove unused import: `common.check_repo_exists`

(F401)